client-v2/src/actions/Clipboard.js

```diff
@@ -52,43 +51,39 @@ function fetchClipboardContentSuccess(clipboardContent: Array<string>) {
   };
 }
 
-function fetchClipboardContent(): ThunkAction {
```

Comment:
I think I need to understand this change a bit more before I can review this. How come we're not fetching the clipboard any more - where is being loaded from now?

Reply:
I _think_ the idea is because Scala is in a position to provide us with this information on our first request, we plant it on window.config and parse it when the app starts up.

Reply:
I thought as much but I just couldn't see where that was happening but maybe it's already happening before this PR?

Reply:
I think that's right! IIRC it happens on the Config instance dropped into the index template, which is extended in this PR to include our options.

Reply:
Cool - this all make much more sense then fankew 👍